### PR TITLE
Fix link for zookeeper archived binary

### PR DESCRIPTION
**Pull request by henrysfx**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,16 +1,16 @@
 # Dockerfile for ZooKeeper
 
-FROM quay.io/signalfuse/maestro-base:alp-3.8-jdk8
+FROM quay.io/signalfuse/signalfx-base:alp-3.8-jdk8
```
Comment (mpetazzoni) [inline Dockerfile:3]: Is this change intentional? That brings much more than just fixing the link to the ZooKeeper tarball.

Comment (henrysfx) [reply Dockerfile:3]: Yes it is intentional and afaict required for the image to work within the GKE realms

Comment (mpetazzoni) [reply Dockerfile:3]: In this case it'd be preferable for those two changes to be in distinct commits and your PR to call out both changes specifically.
